docs/install/mac.md

```diff
@@ -72,6 +72,24 @@ GTKWave is now installed on your macOS. You can start it in your terminal, run:
 gtkwave
 ```
 
+### Building macOS Application
+Ensure you have installed the dependencies as described [above](#installing-dependencies).
+Then run the following command to build the macOS application:
+```shell
+meson setup build \
+    -Dbuild_macos_app=enabled \
+    -Dset_rpath=enabled \
+    -Dintrospection=false \
+    --prefix=/Applications/GTKWave.app \
+    --datadir=/Applications/GTKWave.app/Contents/Resources/share \
+    --bindir=Contents/Resources/bin

```

Comment:
Should we add a shell script for this? I think that would make it easier to build the app and we don't need to remember to update the docs each time the parameters change.